src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -95,6 +99,24 @@ export class NavbarComponent implements OnInit, OnDestroy {
         }
     }
 
+    buildBreadcrumbs(fullURI: string) {
+        this.breadcrumbs = [];
+        // Temporarily restrict routes
+        if (!fullURI.startsWith('/admin') && !fullURI.startsWith('/course-management')) {
+            return;
+        }
+        let path = '';
+        let index = 0;
+        const parts = fullURI.split('/');
+        for (const part of parts) {
+            const crumb = new Breadcrumb();
+            crumb.label = part;
+            path += part + '/';
+            crumb.uri = path;
+            this.breadcrumbs[index++] = crumb;
+        }
```

Comment:
I think you can change this also to something like this:

```suggestion
        this.breadcrumbs: Breadcrumb[] = parts.map(part => {
    path += part + '/';
    return {label: part, uri: path}
})
```